posthog/api/capture.py

```diff
@@ -210,35 +263,17 @@ def get_event(request):
         try:
             distinct_id = _get_distinct_id(event)
         except KeyError:
-            return cors_response(
-                request,
-                generate_exception_response(
```

Comment:
Maybe I'm missing something but aren't we changing the current API behavior by removing those responses?